.azure-pipelines/azure-pipelines-windows.yml

```diff
@@ -2,12 +2,12 @@ jobs:
 
   # Configure, build, install, and test job
   - job: 'windows_build'
-    displayName: 'Windows VS2017'
+    displayName: 'Windows VS2022'
     pool:
-      vmImage: 'vs2017-win2016'
+      vmImage: 'windows-2022'
     timeoutInMinutes: 360
     variables:
-      llvm.version: '11.0.0'
+      llvm.version: '12.0.0'  # 13 bad c. Jan 2022
```

Comment:
13 is good if the vcvarsall.bat all is successful below.

Reply:
yep, got 13 to run. left a commented `vswhere` to help track down vcvarsall in future.

Reply:
That's a handy command to know.